### ISSUE TITLE
Allow easy mouse selection of branch in Alacritty

### ISSUE THREAD

**Issue by kgadek** [PawelLipski/git-machete#508]
# Overview

Currently, this is the output of status:

```console
$ git machete status
  main
  │
  └─issue/abc
  │ │
  │ └─issue/def
  │   │
  │   └─issue/ghi
  │   │
  │   └─issue/jkl
  │
  └─meta/vwx
```

In iTerm, I can double-click on `issue/abc` and it highlights exactly that. However, in Alacritty double-clicking selects `└─issue/abc`. _Not too convenient._

For comparison, `tree` has the following output:

```console
$ tree -L 2
.
├── CONTRIBUTING.md
├── Dockerfile
├── ISSUE_TEMPLATE.md
# …snip…
├── blogs
│   ├── docker-ci-tricks-1
│   ├── docker-ci-tricks-2
│   ├── git-machete-1
│   └── git-machete-2
├── ci
│   ├── checks
│   ├── deb-ppa-test-install
# …snip…
```

<details><summary>I believe here is the easiest possible fix. It aligns <code>git machete status</code> with what <code>tree</code> does by indenting 4 spaces, and adding one space before branch name.</summary>

```diff
diff --git a/git_machete/client.py b/git_machete/client.py
index 8e2b6cc..811c4fe 100644
--- a/git_machete/client.py
+++ b/git_machete/client.py
@@ -936,7 +936,7 @@
             out.write("  ")
             for p in accumulated_path[:-1]:
                 if not p:
-                    out.write("  ")
+                    out.write("    ")
                 else:
                     out.write(colored(f"{utils.get_vertical_bar()} ", edge_color[p]))
             out.write(colored(suffix, edge_color[branch_]))
@@ -972,7 +972,7 @@
                                   f"{dim(commit.short_hash)}  " if opt_list_commits_with_hashes else "", dim(commit.subject),
                                   fp_suffix))
                 elbow_ascii_only: Dict[str, str] = {AnsiEscapeCodes.DIM: "m-", AnsiEscapeCodes.RED: "x-", AnsiEscapeCodes.GREEN: "o-", AnsiEscapeCodes.YELLOW: "?-"}
-                elbow: str = u"└─" if not utils.ascii_only else elbow_ascii_only[edge_color[branch]]
+                elbow: str = u"└── " if not utils.ascii_only else elbow_ascii_only[edge_color[branch]]
                 print_line_prefix(branch, elbow)
             else:
                 if branch != dfs_res[0][0]:

```

Updated output:

```console
$ git machete status
  main
  │
  └── issue/abc
  │ │
  │ └── issue/def
  │     │
  │     └── issue/ghi
  │     │
  │     └── issue/jkl
  │
  └── meta/vwx
```

</details>

I can create a PR, but I'd like to discuss this approach first.

## `git` and `git machete` versions

```console
$ git --version
git version 2.36.0

$ git machete --version
git machete version 3.9.0
```

**Comment by PawelLipski** [comment]
Let's continue on https://github.com/VirtusLab/git-machete/pull/511 then!

**Comment by PawelLipski** [comment]
To be released within v3.10.0 soon.
